dummy.env

```diff
@@ -25,3 +25,13 @@ POSTGRES_DB=wallets
 REDIS_ENABLED=false
 REDIS_HOST=redis
 REDIS_PASS=
+
+
+# for tails server local use only
+
+MSYS_NO_PATHCONV=1
+DOCKERHOST=${APPLICATION_URL-$(docker run --rm --net=host eclipse/che-ip)}
+GENESIS_URL=${GENESIS_URL:-http://$DOCKERHOST:9000/genesis}
```

Comment:
This is a good question - so far we've just been passing genesis files around and not actually exposing a genesis url. Does the tails server require a url? If so we may need to add an endpoint to the agency to server of the genesis file and then point this GENESIS_URL at that endpoint

Reply:
yeah it looks for a URL and I think that is how most of the VON network stuff works. So I think it would be a good idea to do that. But do we need it locally?

Reply:
Well we'd need it in each of our envs + local. Right now we don't have any endpoints that expose our genesis files, adding that endpoint to the agency probably makes the most sense, as it has access to our genesis files